pkgs/tools/package-management/disnix/disnixos/default.nix

```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl, dysnomia, disnix, socat, pkgconfig, getopt }:
 
 stdenv.mkDerivation {
-  name = "disnixos-0.9";
+  name = "disnixos-0.9.1";
```

Comment:
This should be converted to pname at some point and the version number below should be substituted.